src/components/Capture/capture.js

```diff
@@ -66,50 +53,20 @@ class Capture extends Component {
 
   maxAutomaticCaptures = 3
 
-  uploadCaptureToOnfido = () => {
-    this.setState({uploadInProgress: true})
-    const {validCaptures, method, side, token} = this.props
-    const {blob, documentType, id} = validCaptures[0]
-    this.setState({captureId: id})
-
-    if (method === 'document') {
-      const data = { file: blob, type: documentType, side}
-      if (this.state.documentValidations) {
-        data['sdk_validations'] = {'detect_document': 'error', 'detect_glare': 'warn'}
-      }
-      uploadDocument(data, token, this.onApiSuccess, this.onApiError)
-    }
-    else if  (method === 'face') {
-      const data = { file: blob }
-      uploadLivePhoto(data, token, this.onApiSuccess, this.onApiError)
-    }
-  }
-
-  confirmAndProceed = () => {
-    const {method, side, nextStep, actions: {confirmCapture}} = this.props
-    this.clearErrors()
-    confirmCapture({method, id: this.state.captureId, onfidoId: this.state.onfidoId})
-    this.confirmEvent(method, side)
-    nextStep()
-  }
-
-  confirmEvent = (method, side) => {
-    if (method === 'document') {
-      if (side === 'front') events.emit('documentCapture')
-      else if (side === 'back') events.emit('documentBackCapture')
-    }
-    else if (method === 'face') events.emit('faceCapture')
-  }
-
   createCapture(payload) {
-    const { actions, method, side } = this.props
+    const { actions, method, side, nextStep } = this.props
     payload.side = side
```

Comment:
Consider: I just spotted this. Not a problem on your PR, but changing an object inside a function is error prone. I would create a new payload with the side on it, in the `handleDocument` method.